BUILD.md

```diff
@@ -122,7 +122,7 @@ The extension layer tests depend on the
 [SPIRV-Headers repository](https://github.com/KhronosGroup/SPIRV-Headers).
 You must clone the SPIRV-Headers repository
 and build its `install` target. Follow the build instructions in the SPIRV-Headers
-[README.md](https://github.com/KhronosGroup/SPIRV-Headers/blob/master/README.md)
+[README.md](https://github.com/KhronosGroup/SPIRV-Headers/blob/main/README.md)
```

Comment:
SPIRV-* repos already use `main` gslang currently doesn't but will. So might as well do it now.